src/components/Pagination/index.tsx

```diff
@@ -1,42 +1,120 @@
 import * as React from 'react'
 
+import { ChevronIconV2 } from '@root/graphics/ChevronIconV2'
```

Comment:
We should use only icons from the `/icons` directory. Can you make this change?

Reply:
Done!

Reply:
hmmm, did you push your changes?

Reply:
Yes I just triple checked, not sure why it's not showing here...